MetaCell/scicrunch-antibody-registry
# migration mui v5 to mui v6

Problem: Migrate MUI v5 library to MUI v6
Note: Please check [this](https://docs.google.com/document/d/1frGjx6yvGL4PEGxZl4dv8P12YmmzYeu0i2T6w7rbsbU/edit?usp=sharing) documentation to see a brief report of changes made
# Implemented solution

- Update components
- Update mui system props
- update theming

# How to test this PR
In order to migrate to mui v6 and be able to test this pr, we need to follow these steps: 
`npm install @mui/material@latest @mui/system@latest @mui/icons-material@latest @emotion/react@latest @emotion/styled@latest`

for mui-x datagrid component we should: 

1. Update the @mui/x-data-grid version to "6.0.0" in the package.json file.
2. You may need to delete the yarn.lock file and the node_modules folder, then run the “yarn” command again. Afterward, reopen your IDE to ensure the changes are applied.

Note: Mui-x datagrid component "6.0.0" is incompatible with mui v6 version to bypass it instead of `npm install` command we should use yarn


# Sanity checks:

# Breaking changes (select one):

# Possible deployment updates issues (select one):

### Test coverage (select one):

### Documentation (select one):

### Nice to have (if relevant):
